core/components/molecules/column-layout/column-layout.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Automation from '../../_helpers/automation-attribute'
+import transformChildren from '../../_helpers/transform-layout-children'
+import { spacing } from '@auth0/cosmos-tokens'
+
+const gridTemplateColumns = {
+  '1/2 1/2': 'repeat(2, 1fr)',
+  '1/3 1/3 1/3': 'repeat(3, 1fr)',
+  '2/3 1/3': '2fr 1fr',
+  '1/3 2/3': '1fr 2fr',
+  '1/4 1/4 1/4 1/4': `repeat(4, 1fr)`,
+  '2/4 1/4 1/4': '2fr 1fr 1fr',
+  '1/4 2/4 1/4': '1fr 2fr 1fr',
+  '1/4 1/4 2/4': '1fr 1fr 2fr',
+  '3/4 1/4': '3fr 1fr',
+  '1/4 3/4': '1fr 3fr'
+}
+
+const gutterOptions = {
+  'none': "var(--ColumnLayout--None--GridGap)",
+  'condensed': "var(--ColumnLayout--Condensed--GridGap)",
+  'default': "var(--ColumnLayout--Default--GridGap)",
+  'spacious': "var(--ColumnLayout--Spacious--GridGap)"
+}
+
+const marginReset = {
+  'none': "auto",
+  'reset': "0 !important"
+}
+
+
+const ColumnLayout = props => (
+  <ColumnLayout.Element gutter={props.gutter} distribution={props.distribution} {...Automation('column-layout')}>
+    {transformChildren(props, ColumnLayout.Item, 'column-layout.item')}
+  </ColumnLayout.Element>
+)
+
+ColumnLayout.Element = styled.div`
+  --ColumnLayout--None--GridGap: 0;
+  --ColumnLayout--Condensed--GridGap: ${spacing.small};
+  --ColumnLayout--Default--GridGap: ${spacing.medium};
+  --ColumnLayout--Spacious--GridGap: ${spacing.large};
+  --ColumnLayout--Header--Color: ${spacing.large};
+
+  display: grid;
+  grid-gap: ${props => gutterOptions[props.gutter]};
+  grid-template-columns: 1fr;
+  
+  /* Placeholder width media feature until we have global variables for breakpoints */
+  @media (min-width: 768px) {
```

Comment:
Can we make a token for this?

Reply:
Yes we should and will when we tackle the variable system later this year